src/AbstractValue.php

```diff
@@ -315,21 +315,27 @@ protected static function phpVarToNativeXmlRpc($value)
 
         switch (static::getXmlRpcTypeByValue($value)) {
             case self::XMLRPC_TYPE_DATETIME:
+                /** @psalm-suppress MixedArgument */
```

Comment:
No biggie,. but I'd let these land in the baseline

Reply:
Those are not for baseline because value is intentionally mixed. I figured no description is needed.

See https://github.com/vimeo/psalm/issues/7117#issuecomment-991300425

Reply:
Fine by me with this decision: :ship: